examples/ex-offset.cpp

```diff
@@ -13,70 +13,74 @@
 //
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
 
+#include <cmath>
 #include <cstdlib>
 #include <cstring>
 #include <iostream>
-#include <cmath>
 
 #include "RAJA/RAJA.hpp"
 #include "memoryManager.hpp"
 
 /*
  *  Offset example
- *  Example uses a 5-box stencil
- *  to compute the sum of interior boxes.
- *  We assume N x N interior nodes and
- *  a padded edge of zeros for a box of
- *  size (N+2) x (N+2)
  *
- * Input: 
+ *  Example applies a ``five-box stencil" to
+ *  interior cells of a lattice. A five-box stencil
+ *  accumulates values of a box and its four neighbors.
+ *  The resulting values are used to populate the
+ *  interior cells of a second lattice.
+ *  We assume N x N interior nodes and a padded edge
+ *  of zeros for a lattice of size (N + 2) x (N + 2).
+ *
+ *  In the case of N = 2, the first lattice is generated
```

Comment:
Please recall the suggestion I made via email about using N = 3, here in the explanation as well as the example.